concepticondata/conceptlists/Swadesh-1961-100.tsv

```diff
@@ -0,0 +1,101 @@
+Swadesh-1961-100-13	13	grande	big	1202	BIG
+Swadesh-1961-100-14	14	largo	long	1203	LONG
+Swadesh-1961-100-15	15	chico	small	1246	SMALL
+Swadesh-1961-100-16	16	mujer	woman	962	WOMAN
+Swadesh-1961-100-17	17	hombre	man	1554	MAN
+Swadesh-1961-100-18	18	gente	person	683	PERSON
+Swadesh-1961-100-19	19	pez	fish	227	FISH
+Swadesh-1961-100-20	20	pájaro	bird	937	BIRD
+Swadesh-1961-100-21	21	perro	dog	2009	DOG
+Swadesh-1961-100-22	22	piojo	louse	1392	LOUSE
+Swadesh-1961-100-23	23	árbol	tree	906	TREE
+Swadesh-1961-100-24	24	semilla	seed	714	SEED
+Swadesh-1961-100-25	25	hoja	leaf	628	LEAF
+Swadesh-1961-100-26	26	raíz	root	670	ROOT
+Swadesh-1961-100-27	27	corteza	bark	1204	BARK
+Swadesh-1961-100-28	28	piel	skin	763	SKIN
+Swadesh-1961-100-29	29	carne	flesh	2259	FLESH
+Swadesh-1961-100-30	30	sangre	blood	946	BLOOD
+Swadesh-1961-100-31	31	hueso	bone	1394	BONE
+Swadesh-1961-100-32	32	grasa	fat	323	FAT (ORGANIC SUBSTANCE)
+Swadesh-1961-100-33	33	huevo	egg	744	EGG
+Swadesh-1961-100-34	34	cuerno	horn	1393	HORN (ANATOMY)
+Swadesh-1961-100-35	35	cola	tail	1220	TAIL
+Swadesh-1961-100-36	36	pluma	feather	1201	FEATHER
+Swadesh-1961-100-37	37	cabello	hair	1040	HAIR
+Swadesh-1961-100-38	38	cabeza	head	1256	HEAD
+Swadesh-1961-100-39	39	oreja	ear	1247	EAR
+Swadesh-1961-100-40	40	ojo	eye	1248	EYE
+Swadesh-1961-100-41	41	nariz	nose	1221	NOSE
+Swadesh-1961-100-42	42	boca	mouth	674	MOUTH
+Swadesh-1961-100-43	43	diente	tooth	1380	TOOTH
+Swadesh-1961-100-44	44	lengua	tongue	1205	TONGUE
+Swadesh-1961-100-45	45	garra	claw	72	CLAW
+Swadesh-1961-100-46	46	pie	foot	1301	FOOT
+Swadesh-1961-100-47	47	rodilla	knee	1371	KNEE
+Swadesh-1961-100-48	48	mano	hand	1277	HAND
+Swadesh-1961-100-49	49	barriga	belly	1251	BELLY
+Swadesh-1961-100-50	50	cuello	neck	1333	NECK
+Swadesh-1961-100-51	51	teta	breast	1402	BREAST
+Swadesh-1961-100-52	52	corazón	heart	1223	HEART
+Swadesh-1961-100-53	53	hígado	liver	1224	LIVER
+Swadesh-1961-100-54	54	beber	drink	1401	DRINK
+Swadesh-1961-100-55	55	comer	eat	1336	EAT
+Swadesh-1961-100-56	56	morder	bite	1403	BITE
+Swadesh-1961-100-57	57	ver	see	1409	SEE
+Swadesh-1961-100-58	58	oir	hear	1408	HEAR
+Swadesh-1961-100-59	59	saber	know	1410	KNOW (SOMETHING)
+Swadesh-1961-100-60	60	dormir	sleep	1585	SLEEP
+Swadesh-1961-100-61	61	morir	die	1494	DIE
+Swadesh-1961-100-62	62	matar	kill	1417	KILL
+Swadesh-1961-100-63	63	nadar	swim	1439	SWIM
+Swadesh-1961-100-64	64	volar	fly	1441	FLY (MOVE THROUGH AIR)
+Swadesh-1961-100-65	65	andar	walk	1443	WALK
+Swadesh-1961-100-66	66	venir	come	1446	COME
+Swadesh-1961-100-67	67	acostado	lie	1411	LIE (REST)
+Swadesh-1961-100-68	68	sentado	sit	1416	SIT
+Swadesh-1961-100-69	69	parado	stand	1442	STAND
+Swadesh-1961-100-70	70	dar	give	1447	GIVE
+Swadesh-1961-100-71	71	decir	say	1458	SAY
+Swadesh-1961-100-72	72	sol	sun	1343	SUN
+Swadesh-1961-100-73	73	luna	moon	1313	MOON
+Swadesh-1961-100-74	74	estrella	star	1430	STAR
+Swadesh-1961-100-75	75	agua	water	948	WATER
+Swadesh-1961-100-76	76	lluvia	rain	2108	RAINING OR RAIN
```

Comment:
_lluvia_ is usually mapped to RAIN (PRECIPITATION), except for [Heggarty-2005-150-81](https://concepticon.clld.org/values/Heggarty-2005-150-81). I'd follow the mapping of the majority of the lists and remap this one.